src/scripts/gym/Champion.ts

```diff
@@ -47,7 +47,7 @@ class Champion extends Gym {
 
 gymList['Champion Blue'] = new Champion(
```

Comment:
Should this not also be updated to reflect the change?
Otherwise it seems like each of the elites are being referred to 3 different ways:
`Blue`, `Champion Blue`, `Kanto Champion Blue`
```suggestion
gymList['Kanto Champion Blue'] = new Champion(
```

Reply:
Done